### DISCUSSION TITLE
What's the reason to roll back supported Isaac Lab Version?

### DISCUSSION THREAD

**Discussion by deyuf** [abizovnuralem/go2_omniverse#38] [Q&A]
Hi @abizovnuralem,

In the newest commit, I see that the support version of Isac Sim is changed from 4.0 to Orbit. Is there any reason for this change?

Best,

**Comment by abizovnuralem** [comment]
Hello, thanks for your question , please refer to this https://github.com/abizovnuralem/go2_omniverse/issues/30 and https://github.com/abizovnuralem/go2_omniverse/issues/30#issuecomment-2323114485
